README.md

```diff
@@ -2,97 +2,120 @@
   <img src="media/eslint-webcomponents.svg" alt="Eslint + WebComponents" width="450" height="175" />
 </div>
 
-# eslint-plugin-wc
-
-ESLint plugin for Web Components.
+# `eslint-plugin-wc`
 
 [![npm version](https://img.shields.io/npm/v/eslint-plugin-wc.svg?style=flat)](https://npmjs.org/package/eslint-plugin-wc 'View this project on npm')
 [![Build Status](https://travis-ci.com/43081j/eslint-plugin-wc.svg?branch=master)](https://travis-ci.com/43081j/eslint-plugin-wc)
 [![Coverage Status](https://coveralls.io/repos/github/43081j/eslint-plugin-wc/badge.svg?branch=master)](https://coveralls.io/github/43081j/eslint-plugin-wc?branch=master)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
+> ESLint plugin for [Web Components](https://www.webcomponents.org/introduction).
+
 ## Install
 
-```
-$ npm i -D eslint eslint-plugin-wc
+Assuming you already have ESLint installed, run:
+
+```sh
+# npm
+npm install eslint-plugin-wc --save-dev
+
+# yarn
+yarn add eslint-plugin-wc --dev
 ```
 
 ## Usage
 
-Add `wc` to the plugins section of your `.eslintrc` file:
+Then extend the recommended eslint config:
 
-```json
+```js
 {
-  "plugins": ["wc"]
+  "extends": [
+    // ...
+    "plugin:wc/recommended"
+  ]
 }
 ```
 
-Configure your rules like so:
+You should also specify settings that will be shared across all the plugin rules. ([More about eslint shared settings](https://eslint.org/docs/user-guide/configuring/configuration-files#adding-shared-settings))
 
-```json
+```jsonc
 {
-  "rules": {
-    "wc/rule-name": "error"
+  "settings": {
+    "wc": {
+      "elementBaseClasses": ["LitElement"] // Recognize `LitElement` as a Custom Element base class
+    }
   }
 }
 ```
 
-## Configuration
+### Custom Configuration
 
-You may also extend the recommended configuration like so:
+If you want more fine-grained configuration, you can instead add a snippet like this to your ESLint configuration file:
 
-```json
+```js
 {
-  "extends": ["plugin:wc/recommended"],
-  "env": {"browser": true}
+  "plugins": [
+    // ...
+    "wc"
+  ],
+  "rules": {
+    // ...
+    "wc/no-invalid-element-name": "error",
+    "wc/no-typos": "warn"
+  }
 }
 ```
 
-Or the best practice configuration which extends this to include all
-best practice rules:
+## Supported Rules
 
-```json
-{
-  "extends": ["plugin:wc/best-practice"],
-  "env": {"browser": true}
-}
-```
+<!-- TODO: Generate with `md-magic` -->
 
-### Custom settings
+### Possible Errors (Recommended)
 
-Settings are configured through a `settings` option in your eslintrc file.
+| Rule                                                                    | Description                                           | Fixable? |
```

Comment:
can we turn this into a bullet list while we're here? so its the same as the lit plugin, lot easier to maintain

Reply:
I was going to leave this here and add md-magic to generate the table automatically in a follow up pr

Reply:
tbh its never been filled in other than the description and you can infer that from the name usually. i'd keep it simple as a plain list personally

Reply:
Okay, will push one later